workflow/rocoto_funcs/ungrib_lbc.py

```diff
@@ -8,16 +8,28 @@ def ungrib_lbc(xmlFile, expdir, do_ensemble=False):
   offset=int(os.getenv('LBC_OFFSET','6'))
   length=int(os.getenv('LBC_LENGTH','12'))
   interval=int(os.getenv('LBC_INTERVAL','3'))
+  fhr=os.getenv('FCST_LENGTH','6')
```

Comment:
The `fhr` variable is not used anywhere in this script.

Reply:
Removed. Thanks!